File: app/src/main/java/com/example/yc/zhihudaily/Presenter/Contract.java
package com.example.yc.zhihudaily.Presenter;

import com.example.yc.zhihudaily.Model.Datas;

public class Contract {

    public interface ViewPresenter {
        void initView();
        void updateView();
        void setData(Datas datas);
    }

    public interface ListPresenter{
        void getData();
        void loadMore(String date);
    }
}

File: app/src/main/java/com/example/yc/zhihudaily/Presenter/Contract.kt
package com.example.yc.zhihudaily.Presenter

import com.example.yc.zhihudaily.Model.Datas

class Contract {

    interface ViewPresenter {
        fun initView()
        fun refreshView()
        fun updateView()
        fun setData(datas: Datas)
    }

    interface ListPresenter {
        fun getData()
        fun refreshData()
        fun loadMore(date: String)
    }
}
